### DISCUSSION TITLE
Enhancing Open Source Collaboration in @carbon/ibm-products

### DISCUSSION THREAD

**Discussion by saogregl** [carbon-design-system/ibm-products#6627] [General] [answered]
Hi, I’m Lucas! I’m a hobby developer who’s been using @carbon/react and @carbon/ibm-products for a while, mainly to build internal tools at my job as a mechanical engineer. Although this isn’t part of my formal role, I love working with the Carbon Design System and use it for most of my projects. I’m not an expert in UI/UX, but I’d love to contribute by tackling issues and learning more about the process. Huge thanks to IBM for open-sourcing such an amazing design system!

### Context

The @carbon-design-system/ibm-products repository is, in my opinion, a very important resource for users, while the @carbon-design-system/carbon monorepo provides essential primitives, this repository effectively implements patterns widely used in real-world applications. While publicly available, much of the reasoning, design explorations, and resources remain within IBM’s internal platforms. This includes:

- A product gallery showcasing real-world use of the design system.
- Design exploration documents explaining the rationale behind decisions.

These resources are incredibly valuable but currently unavailable to the wider open-source community. While understandable, this makes it harder for external contributors to:

1. Understand the design system’s goals and roadmap.
2. Gain inspiration from existing implementations.
3. Actively contribute to its evolution.

This can limit adoption and innovation within the open-source community.

I've created this discussion (and I hope this is the right place for this) to suggest a few things: 

1. **Publicly Share Select Resources**:
   - A public version of the product gallery to inspire the community.
   - Sanitized design exploration documents focusing on the “why” behind decisions.

2. **Facilitate Collaboration**:
   - Open forums or discussions to gather community feedback.
   - A partially open roadmap to guide contributors on upcoming work.

I understand that some resources need to remain internal, but curated materials and increased collaboration could significantly benefit both IBM and the community. 

Thanks you all very much!

**Comment by sstrubberg** [comment, marked as answer]
@saogregl thank you so much for raising this discussion. I'm so happy that not only has Carbon Core been helpful for you and your projects, but Carbon for IBM Products as well. 2025 is shaping up to be a crazy year. Something that we're wrestling with internally is the future of Carbon for IBM Products. What role does it play in the ecosystem? Is it open or inner-source? Hearing from you and others like you that benefit from C4IBMP definitely help us shape our roadmap and decisions we make. We'll keep you posted on where things are headed. Hopefully there's a path where we can more openly share this type of work in the future.
